tests/functional/lti_certification/v13/conftest.py

```diff
@@ -63,18 +63,18 @@ def application_instance(lti_registration):
 
 
 @pytest.fixture(autouse=True)
-def jwk_endpoint_intercept(lti_registration, jwt_public_key):
+def mock_http_jwt_endpoint(lti_registration, jwt_public_key):
+    """Mock the response of the platform's JWT HTTP enpoint."""
     jwk = {"keys": [jwt_public_key]}
 
-    httpretty.register_uri(
-        method="GET",
-        uri=lti_registration.key_set_url,
-        body=json.dumps(jwk),
-    )
+    with httpretty.enabled():
+        httpretty.register_uri(
+            method="GET",
```

Comment:
To prove this is actually in effect now changing GET to POST or anything else will result if failing tests.